addons/l10n_it_edi_sdicoop/models/account_invoice.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding:utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo import fields, models
+
+
+_logger = logging.getLogger(__name__)
+
+DEFAULT_FACTUR_ITALIAN_DATE_FORMAT = '%Y-%m-%d'
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_it_edi_transaction = fields.Char()
+    l10n_it_edi_attachment_id = fields.Many2one('ir.attachment')
```

Comment:
So we want this separate attachment for the case where we send but have not received the outcome yet?  (just to reconfirm)

Reply:
Yes, we will never receive the outcome straight away. It will be done in the second step, but we need a reference to the attachment, so that we can return it when we received the positive outcome.